client/src/theme/themes.ts

```diff
@@ -27,6 +27,30 @@ export const themes: { [key: string]: ThemeType } = {
     // Typography
     'bs-font-sans-serif': 'system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"',
   },
+  construction: {
+    'bs-body-bg': '#FFA500',
+    'bs-body-color': '#FFFF00',
+    'bs-btn-primary-bg': '#00FF00',
+    'bs-btn-primary-color': '#ffffff',
+    'bs-link-color': '#0000FF',
+    'bs-link-hover-color': '#789',
+    'bs-background-color': '#000000',
+    'sidebar-text-color': '#ffffff',
+    'sidebar-text-color-active': 'grey',
+    // Typography
+    'bs-font-sans-serif': 'system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"',
+  }, 
+  medicine: {
+    'bs-body-bg': '#1E90FF',
+    'bs-body-color': '#FFFFFF', 
+    'bs-btn-primary-bg': '#F5F5F5', 
+    'bs-btn-primary-color': '#000000',
+    'bs-link-color': '#FF0000', 
+    'bs-link-hover-color': '#800000',
+    'bs-background-color': '#FFFFFF',
+    'sidebar-bg-color': '#FFFFFF',
+    'sidebar-text-color': '#000000',
+    'sidebar-text-color-active': '#FF1493'
+  }
```

Comment:
Are you able to add color themes for more industries?

Reply:
Yes. It is dynamic so if anybody has themes they are interested in, simply add it to the file like the above examples and it should show up.

Reply:
I can add more later but paste here if you have any immediate ideas you want before this PR is merged.